src/h2olog/h2olog.cc

```diff
@@ -232,11 +233,12 @@ int main(int argc, char **argv)
     int debug = 0;
     bool preserve_root = false;
     bool list_usdts = false;
+    bool appdata = false;
```

Comment:
```suggestion
    bool include_appdata = false;
```

Boolean variables above use verb + object, it'd be good to follow the convention.